go.mod

```diff
@@ -1,5 +1,8 @@
-module github.com/containerd/btrfs
+module github.com/containerd/btrfs/v2
 
-go 1.15
+go 1.16
 
-require github.com/pkg/errors v0.9.1
+require (
+	github.com/dennwc/ioctl v1.0.0
```

Comment:
This looks like a really small dependency; do you think it would make sense to include it in some other dependency (or include it in this codebase?)

Reply:
done

Reply:
Thanks!